operatorconfig/driverconfig/powerflex/v2.11.0/controller.yaml

```diff
@@ -168,7 +168,7 @@ spec:
             - name: socket-dir
               mountPath: /var/run/csi
         - name: csi-external-health-monitor-controller
-          image: registry.k8s.io/sig-storage/csi-external-health-monitor-controller:v0.11.0
+          image: registry.k8s.io/sig-storage/csi-external-health-monitor-controller:v0.12.1
```

Comment:
https://github.com/dell/csm-operator/blob/08e1f66b93692f605926e41fa51a1a2a42c7d6a1/tests/config/driverconfig/powerflex/v2.11.0/controller.yaml#L165 - Lot of test files are there. it needs updates?

Reply:
Discussing with @JacobGros and @abhi16394 about whether these changes are required.

Reply:
Updated test files to use new sidecar image.